product_docs/docs/pem/8.1.0/pem_inst_guide_linux/04_installing_postgres_enterprise_manager/06_configuring_the_pem_server_to_use_kerberos_authentication.mdx

```diff
@@ -34,140 +35,138 @@ $ sudo kadmin.local -q "addprinc -randkey postgres/<HOSTNAME_OF_PEM_SERVER>"
 Where, `HOSTNAME_OF_PEM_SERVER` must contain the `REALM` of the `Kerberos Server`. For example - You can specify `pemdb.edbpem.org` as the hostname of PEM Server with `edbpem.org` as the `REALM`.
 
 !!! Note
-    
     -   If the PEM Web Application and the PEM Backend Database Server are on different machines, then hostname for both will be different.
 
 ## 3. Extract Key Tables from Kerberos Server
 
 a.  Extract the Key Tables from Kerberos for the PEM Web Application and the PEM Backend Database Server:
 
-   ```text
-   sudo kadmin.local "ktadd -k <NAME_OF_PEM_WEB_FILE>.keytab HTTP/<HOSTNAME_OF_PEM_SERVER>"
-   sudo kadmin.local "ktadd -k <NAME_OF_PEM_DB_FILE>.keytab postgres/<HOSTNAME_OF_PEM_SERVER>"
-   ```
+```text
```

Comment:
Doesn't this mess up the indenting? This code should be part of the a. item rather than flush to the right, I would think.

Reply:
It would, if this was a real list. But it isn't, so it doesn't do anything other than make it less likely someone will THINK it is indented.

Wouldn't hurt to make it a real list though, and then indent